### ISSUE TITLE
LSL Support

### ISSUE THREAD

**Issue by winterwolf** [geany/geany#2183]
I have been dreaming for a long time to see the [LSL](http://wiki.secondlife.com/wiki/LSL_Portal) support in Geany. [There are](http://wiki.secondlife.com/wiki/LSL_Alternate_Editors) so many editors supporting this language, but unfortunately Geany is not on this list.

**Comment by elextr** [comment]
Feel free to contribute a [custom filetype](https://www.geany.org/manual/current/index.html#custom-filetypes), and to add syntax highlighting to the [Scintilla](https://www.scintilla.org/index.html) editor widget Geany uses and symbol parsing to [uctags](https://github.com/universal-ctags/ctags) which is used by Geany for symbol parsing.

**Comment by winterwolf** [comment]
Here are ([1](http://adammarker.org/shill/scite0/lsl.api), [2](http://adammarker.org/shill/scite0/lsl.properties)) syntax highlighting files for SciTE (I hope it is the same as Scintilla). Will they work in Geany?

**Comment by codebrainz** [comment]
No, Geany doesn't read those files, but it shows that you can use the C/C++ lexer using custom filetypes as mentioned by @elextr. At the very least you can get usable syntax highlighting.
